File: programme/area-of-trapezoid/area-of-trapezoid.go
// Write a programmeto compute area of trapezoid
package main

import "fmt"

func main() {
	var a, b, h float64
	fmt.Print("Enter the value of a : ")
	fmt.Scan(&a)
	fmt.Print("Enter the value of b : ")
	fmt.Scan(&b)
	fmt.Print("Enter the value of h : ")
	fmt.Scan(&h)
	fmt.Println("\nThe area of the trapezoid is :", (a+b)*h/2)
}

File: programme/area-of-trapezoid/area-of-trapezoid.js
/* 
Write a program that:

- Takes the length of two bases and height
- Uses the formula to calculate the area
- Return the area

Area of a trapezoid = ( base 1 + base 2 ) * height / 2
*/

function areaTrapezoid(base1, base2, height) {
  return ((base1 + base2) * height) / 2;
}

const base1 = 5;
const base2 = 10;
const height = 7;
console.log("Input  :", base1, base2, height);
console.log("Output :", areaTrapezoid(base1, base2, height));